src/styles/blocks/star.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  width: 96px;
+  margin-left: 18px;
+}
+
+.stars__star {
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
```

Comment:
The same issue with the absolute units. It's better to use relative units for layout-related properties.